usr/lib/linuxmint/mintUpdate/mintUpdate.py

```diff
@@ -907,19 +907,19 @@ def run(self):
                                 infobar_title = _("Please switch to another mirror")
                                 infobar_message = _("%s is unreachable.") % mirror_url
                         elif mint_timestamp is not None:
-                            mint_date = datetime.datetime.fromtimestamp(mint_timestamp)
-                            now = datetime.datetime.now()
+                            mint_date = datetime.fromtimestamp(mint_timestamp)
+                            now = datetime.now()
```

Comment:
This is not related to this PR, it shouldn't be here. Also it's incompatible with master.